Configurations/unix-Makefile.tmpl

```diff
@@ -374,16 +374,16 @@ uninstall: uninstall_docs uninstall_sw
 libclean:
 	@set -e; for s in $(SHLIB_INFO); do \
 		if [ "$$s" = ";" ]; then continue; fi; \
-		s1=`echo "$$s" | cut -f1 -d";"`; \
-		s2=`echo "$$s" | cut -f2 -d";"`; \
-		$(ECHO) $(RM) $$s1; {- output_off() unless windowsdll(); "" -}\
-		$(RM) apps/$$s1; \
-		$(RM) test/$$s1; \
-		$(RM) fuzz/$$s1; {- output_on() unless windowsdll(); "" -}\
-		$(RM) $$s1; \
+		s1="`echo "$$s" | cut -f1 -d";"`"; \
+		s2="`echo "$$s" | cut -f2 -d";"`"; \
```

Comment:
I wondered about the quotes in a quoted string (and why they dont have to be escaped).
Do you think adding double quotes around back-tick is necessary here?
```
$ x1=`echo "test aa   bb"`
$ echo "$x1"
test aa   bb
```
Does the expansion in the make file work different?

Reply:
You're right, it doesn't seem to be strictly necessary in this case. I developed a habit to put all backtick (though I only have experience with the `$(...)` syntax) code around quotes, because in certain other context it's necessary, and this removes the ambiguity.